web_requester.go

```diff
@@ -92,14 +92,15 @@ type WebRequesterFactory struct {
 	Hosts                  []string          `yaml:"Hosts"`
 	Headers                map[string]string `yaml:"Headers"`
 	Body                   string            `yaml:"Body"`
+	BodyPath               string            `yaml:"BodyPath"`
```

Comment:
Let's call it `BodyFile` instead of `BodyPath`. Path implies directory.

Reply:
Done.